.github/workflows/ci.yml

```diff
@@ -51,3 +51,19 @@ jobs:
       - name: Check (Linux)
         if: runner.os == 'Linux'
         run: ./gradlew $GRADLE_ARGS check
+
+      - name: PublishDocumentation
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          ./gradlew dokkaHtml
+          git config user.email CONX_GH_PAGES_USER_EMAIL
+          git config user.name CONX_GH_PAGES_USER_NAME
```

Comment:
These need to be variables, not static text, right?

Reply:
Ah. Yeah, probably.